modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/enumerations.scala

```diff
@@ -158,4 +158,30 @@ object enumerations {
       "CUSTOM_MASK" -> "Custom Mask"
     )
   }
+
+  object disperser {
+    val GmosSDisperser: Map[String, String] = Map(
+      "MIRROR"      -> "Mirror",
+      "B1200_G5321" -> "B1200",
+      "R831_G5322"  -> "R831",
+      "B600_G5323"  -> "B600",
+      "R600_G5324"  -> "R600",
+      "R400_G5325"  -> "R400",
+      "R150_G5326"  -> "R150"
+    )
+
+    // TODO: Do we still need to include the obsolete filters here?
```

Comment:
I don't think we need them as you wouldn't load sequneces with old filters

Reply:
That's what I figured. I didn't include them since then the mapping wouldn't be injective.